plover/dictionary/rtfcre_dict.py

```diff
@@ -13,326 +13,134 @@
 
 """
 
-import codecs
-import inspect
 import re
+import string
 
+from plover.formatting import ATOM_RE
 from plover.steno import normalize_steno
 from plover.steno_dictionary import StenoDictionary
-# TODO: Move dictionary format somewhere more canonical than formatting.
-from plover.formatting import ATOM_RE
-
-
-# A regular expression to capture an individual entry in the dictionary.
-DICT_ENTRY_PATTERN = re.compile(r'(?s)(?<!\\){\\\*\\cxs (?P<steno>[^}]+)}' + 
-                                r'(?P<translation>.*?)(?:(?<!\\)(?:\r\n|\n))*?'+
-                                r'(?=(?:(?<!\\){\\\*\\cxs [^}]+})|' +
-                                r'(?:(?:(?<!\\)(?:\r\n|\n)\s*)*}\s*\Z))')
 
-class TranslationConverter:
-    """Convert an RTF/CRE translation into plover's internal format."""
-    
-    def __init__(self, styles={}):
-        self.styles = styles
-        
-        def linenumber(f):
-            return f[1].__code__.co_firstlineno
-        
-        handler_funcs = inspect.getmembers(self, inspect.ismethod)
-        handler_funcs.sort(key=linenumber)
-        handlers = [self._make_re_handler(f.__doc__, f)
-                    for name, f in handler_funcs 
-                    if name.startswith('_re_handle_')]
-        handlers.append(self._match_nested_command_group)
-        def handler(s, pos):
-            for handler in handlers:
-                result = handler(s, pos)
-                if result:
-                    return result
+from .rtfcre_parse import parse_rtfcre
+
+
+HEADER = (r'{\rtf1\ansi{\*\cxrev100}'
+          r'\cxdict{\*\cxsystem Plover}'
```

Comment:
I think the `\cxsystem` here should include the version number

Reply:
Done.